CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# 0.15.0 (17 May 2022)
+
+This is a major release that adds new features and fixes issues. Specifically:
+* `BigInt` and `BigInt`-based `JSTypedArray` types are now supported. Now, when passing `Int64` values from Swift,
+they will be mapped to `BigInt` values on the JavaScript side. 
+* The `constructor` property on `JSBridgedClass` is now an Optional, which allows bridging JavaScript classes that aren't
+available in every browser or environment.
+* JavaScriptKit runtime files are now supplied as SwiftPM resources. This allows us to resolve a long-standing issue
+in `carton` that could lead to a version mismatch between JavaScriptKit dependency in `Package.swift` or
+`Package.resolved` and carton’s bundled JavaScriptKit runtime version.
+* The `JSSymbol` type has been added, enabling support for [JavaScript `Symbol` values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol), including accessing `Symbol`-keyed properties on objects.
+
```

Comment:
```suggestion

**Source breaking changes**

`UInt64.jsValue` and `Int64.jsValue`, which are a part of `JavaScriptKit` module, has been moved into `JavaScriptBigIntSupport` module since their implementation has been changed to require [JS-BigInt-integration](https://github.com/WebAssembly/JS-BigInt-integration) to avoid implicit cast from 64-bit integer to JS `Number`. 

If you want to keep the behavior so far, please cast the 64-bit integer values to `Double`.

```